### PR TITLE
Fix errors.

### PR DESCRIPTION
**Pull request by aditisingh2362**
Fixed grammatical errors and changed long phrases with a single word to make docs more easier to read.

### CODE REVIEW

File: docs/gsoc/2020/ideas.rst [hunk 1]
```diff
@@ -5,13 +5,13 @@ Introduction
 ------------
 
 This is the Google Summer of Code 2020 (GSoC'20) ideas page for ``uarray``,
-``unumpy`` and ``udiff``. The ``uarray`` library is is a backend mechanism
```
Comment (hameerabbasi) [inline docs/gsoc/2020/ideas.rst]: Can you revert just this file, I'm pretty sure we aren't allowed to change it after GSoC begins and we get our ideas in.

Comment (aditisingh2362) [reply docs/gsoc/2020/ideas.rst]: Sure.

Comment (prasunanand) [reply docs/gsoc/2020/ideas.rst]: @hameerabbasi : I think these changes are fine.

Comment (hameerabbasi) [reply docs/gsoc/2020/ideas.rst]: The changes themselves are fine, it's just that we've submitted our organization ideas and we risk disqualification if we modify the ideas now.

Comment (hameerabbasi) [reply docs/gsoc/2020/ideas.rst]: @python-gsoc Could you comment on this?

Comment (prasunanand) [reply docs/gsoc/2020/ideas.rst]: ok.

**Pull request merged**